changelog/unreleased/features/2000--projection-transform.md

```diff
@@ -0,0 +1 @@
+A new transform step, `projection`, is added to VAST.
```

Comment:
Please rewrite in active-voice, present-tense. What does this mean for the user? What can the user do now that they were not able to do before?